packages/@okta/vuepress-site/docs/reference/rest/index.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Test the Okta REST APIs with Postman
+language: rest
+integration: back-end
+icon: code-rest
+meta:
+  - name: description
+    content: Get started with Okta REST APIs and learn how to import a collection and send requests in Postman.
+---
+
+A great way to learn an Application Programming Interface (API) is to issue requests and inspect the responses. You can use Okta Postman collections to learn how to incorporate Okta APIs into your workflow. To use these collections, you need to:
```

Comment:
This is something we change on the product doc side at Globalization team's request; may be useful in the future if dev docs get translated. An intro stem needs to be a complete sentence because they can't resolve the fragment with the list items. So not really incorrect, but a good practice to get in the habit.